### PR TITLE
Added landing message issue #id1

### PR DESCRIPTION
**Pull request by drashish77**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
- 
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Potionfy</title>
+  </head>
+  <body>
+    <h1>Welcome to potionfy</h1>
```
Comment (alex-coderr) [inline index.html]: Can you remove 'Enjoy your stay string and make the welcome to pontify!? string red?

**Comment by alex-coderr** [comment]
yes, sure

File: index.html (hunk 1 above)
Comment (drashish77) [reply index.html]: Yeah, sure!

**Review by alex-coderr — CHANGES_REQUESTED** [review]
Please, change the code as per my comment on the review

**Review by alex-coderr — APPROVED** [review]

**Review by alex-coderr — APPROVED** [review]
Well done Alex!!

**Pull request merged**